### PR TITLE
Add denommetadata check in rollapp_genesis_event_test

### PR DESCRIPTION
**Pull request by trinitys7**
Partly solved #58

### CODE REVIEW

File: tests/rollapp_genesis_event_test.go [hunk 1]
```diff
@@ -215,4 +214,26 @@ func TestRollappGenesisEvent_EVM(t *testing.T) {
 	require.NoError(t, err)
 
 	testutil.AssertBalance(t, ctx, rollapp1, escrowAddress, rollapp1.Config().Denom, genesisCoin.Amount)
+
+	denommetadata, err := dymension.GetNode().QueryDenomMetadata(ctx, genesisCoin.Denom)
```
Comment (mtsitrin) [inline tests/rollapp_genesis_event_test.go:218]: @trinitys7 `vfc` created by hook should be tested here as well? or a separate test?

Comment (trinitys7) [reply tests/rollapp_genesis_event_test.go:218]: I think it's kinda hard for e2e-test to query a Solidity contract. Do you have any idea about how we do that?

Comment (mtsitrin) [reply tests/rollapp_genesis_event_test.go:218]: I thought `vfc` has a query command, but apparently it doesn't
if u have access to the `x/evm` keeper, u can use `k.GetVirtualFrontierBankContractAddressByDenom`

Comment (mtsitrin) [reply tests/rollapp_genesis_event_test.go:218]: added separate issue for it https://github.com/dymensionxyz/e2e-tests/issues/71

**Review by mtsitrin — APPROVED** [review]

**Pull request merged**